HabbitLogger.antonderegt/HabbitLogger/Program.cs

```diff
@@ -0,0 +1,254 @@
+        @"
+            SELECT count(*)
+            FROM habit
+        ";
+
+        long count = (long)command.ExecuteScalar()!;
+
+        if (count == 0)
+        {
+            command.CommandText =
+            @"
+                INSERT INTO habit
+                VALUES (1, 'Drink water', 15, 'cup(s)'),
+                       (2, 'Wake up', 0, 'times');
+            ";
+            command.ExecuteNonQuery();
+        }
+    }
+
+    static void ShowMenu()
+    {
+        Console.Clear();
+        Console.WriteLine("MAIN MENU\n");
+        Console.WriteLine("What would you like to do?\n");
+        Console.WriteLine("Type 0 to Close Application.");
+        Console.WriteLine("Type 1 to View All Records.");
+        Console.WriteLine("Type 2 to Insert Record.");
+        Console.WriteLine("Type 3 to Delete Record.");
+        Console.WriteLine("Type 4 to Update Record.");
+
+        Console.Write("\nSelect option: ");
+    }
+
+    bool ProcessInput()
+    {
+        string? input = Console.ReadLine();
+
+        if (int.TryParse(input, out int number))
+        {
+            switch (number)
+            {
+                case 0:
+                    return false;
+                case 1:
+                    ShowAllRecords();
+                    Console.WriteLine("\nPress enter to return to the menu...");
+                    Console.ReadLine();
+                    return true;
+                case 2:
+                    InsertRecord();
+                    return true;
+                case 3:
+                    DeleteRecord();
+                    return true;
+                case 4:
+                    UpdateRecord();
+                    return true;
+                default:
+                    Console.WriteLine("\nUnknown action, press enter to try again...");
+                    Console.ReadLine();
+                    return true;
+            }
+        }
+        else
+        {
+            Console.WriteLine("\nPlease enter a number, press enter to try again...");
+            Console.ReadLine();
+            return true;
+        }
+    }
+
+    void ShowAllRecords()
+    {
+        Console.Clear();
+        Console.WriteLine("Your habits:\n");
+        using var connection = new SqliteConnection(connectionString);
+        connection.Open();
+        SqliteCommand command = connection.CreateCommand();
+
+        command.CommandText =
+        @"
+            SELECT name, quantity, unitOfMeassure
+            FROM habit
+        ";
+
+        using var reader = command.ExecuteReader();
+
+        while (reader.Read())
+        {
+            Console.WriteLine($"{reader.GetString(0)}: {reader.GetString(1)} {reader.GetString(2)}.");
+        }
+    }
+
+    void InsertRecord()
+    {
+        Console.Clear();
+        using var connection = new SqliteConnection(connectionString);
+        connection.Open();
+        SqliteCommand command = connection.CreateCommand();
+
+        Console.Write("New habit: ");

```

Comment:
Hey there i think you forgat to ask the quantity for the record to user. 
It is hard coded as 0 for all records.